base/windows/CharsStatusWindow.js

```diff
@@ -81,23 +81,37 @@ export class CharsStatusWindow {
             let info_sprites_obj = {};
             const char = chars_list[i];
             const base_x_pos =  i * (WIDTH_PER_CHAR + SEPARATOR_WIDTH) + INITIAL_PADDING_X;
+
+            info_sprites_obj.group = this.game.add.group();
+            this.status_window.group.add(info_sprites_obj.group)
+
             info_sprites_obj.name = this.status_window.set_text_in_position(char.name, base_x_pos, this.name_y, false, false, this.status_window.font_color, this.compact);
             let y_pos = this.name_y + numbers.FONT_SIZE;
+            if(info_sprites_obj.name.text_bg){
```

Comment:
You don't need to worry about `text_bg` since we are sure that we're not using it.